js/index.js

```diff
@@ -1,6 +1,49 @@
 // Iteration 1: Names and Input
 //
+let hacker1 = 'Jose';
+let hacker2 = 'Farzin';
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
+
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log(
+    'The driver has the longest name, it has ' + hacker1.length + ' characters.'
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    'It seems that the navigator has the longest name, it has ' +
+      hacker2.length +
+      ' characters.'
+  );
+} else {
+  console.log(
+    'Wow, you both have equally long names, ' + hacker1.length + ' characters!'
+  );
+}
 
 // Iteration 3: Loops
+
+console.log(hacker1.toUpperCase().split(''));
+
+console.log(hacker2.split('').reverse().join(''));
+
```

Comment:
The idea was to use loops on that iteration, you get almost all the correct should print a string and not an Array